products-service/src/main/java/com/petclinic/products/presentationlayer/products/ProductRequestModel.java

```diff
@@ -15,6 +16,7 @@ public class ProductRequestModel {
     private String productName;
     private String productDescription;
     private Double productSalePrice;
-    private String productType;
+    private Double averageRating;
```

Comment:
I removed averageRating from the ProductRequestModel in [my PR](https://github.com/cgerard321/champlain_petclinic/pull/944) to fix any averageRating overrides. If you could remove the averageRating field from requestModel please.

Reply:
Thanks for the review Victor. I think my main wasn't updated when I was coding but I fixed and didn't have the fixed bug you did but I fixed and removed the averageRating